docs/user/orion.rst

```diff
@@ -10,77 +10,82 @@ Orion
 Overview
 --------
 
-The `Orion plugin <https://github.com/ls1intum/Orion>`_ for `IntelliJ <https://www.jetbrains.com/idea/>`_ supports students, teaching assistants (tutors), and instructors with the conduction of programming exercises. It integrates Artemis in IntelliJ and automates the download and upload of exercises.
+The `Orion plugin <https://github.com/ls1intum/Orion>`_ for `IntelliJ <https://www.jetbrains.com/idea/>`_ supports students, teaching assistants (tutors), and instructors with the conduction of programming exercises. It integrates Artemis in IntelliJ and automates the download (clone) and upload (push) of exercises.
 
 Installation
 ^^^^^^^^^^^^
 
-Orion is installed like every other plugin via IntelliJ's marketplace. Alternatively, the latest build can be found in `Orion's GitHub repository <https://github.com/ls1intum/Orion>`_ as described there.
+Orion is installed like every other plugin via IntelliJ's marketplace. Alternatively, although not usually needed, it is possible to install builds directly from `Orion's GitHub repository <https://github.com/ls1intum/Orion>`_. The installation process is described in the `readme <https://github.com/ls1intum/Orion#testing-of-pull-requests>`_.
 
 General Usage
 ^^^^^^^^^^^^^
 
 After installation, Orion provides a custom tool window at *View -> Tool Windows -> Artemis*. This window contains an integrated browser displaying Artemis.
 
-At the top of the tool window is a back button |back-button|. Clicking it will return to the initial page, that is:
+At the top of the tool window is a back button |back-button|. Clicking it returns to the initial page, that is:
 
-- The exercise details page if an exercise is opened as a student
-- The exercise edit in editor page if an exercise is opened as an instructor
-- The assessment dashboard if an exercise is opened as a tutor, or, if a submission is downloaded, the assessment details page of that submission
-- The Artemis home page if no Artemis project is opened
+- The exercise details page if an exercise is opened as a student.
```

Comment:
Personally, I downloaded the artifact from the doc action and opened it locally. The page at the link is also broken for me. But since also the content list is broken (and that is purely autogenerated), I assume its a problem on their side.

Reply:
Yes, thanks. That works.